test/e2eexternal/setup.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+## Process command line flags ##
+source ${SCRIPTS_DIR}/lib/shflags
+DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
+DEFINE_string 'deploytool' 'operator' 'Tool to use for deploying (operator/helm)'
+DEFINE_string 'deploytool_broker_args' '' 'Any extra arguments to pass to the deploytool when deploying the broker'
+DEFINE_string 'deploytool_submariner_args' '' 'Any extra arguments to pass to the deploytool when deploying submariner'
+DEFINE_boolean 'globalnet' false "Deploy with operlapping CIDRs (set to 'true' to enable)"
+DEFINE_boolean 'service_discovery' false "Enable multicluster service discovery (set to 'true' to enable)"
+DEFINE_string 'timeout' '5m' "Timeout flag to pass to kubectl when waiting (e.g. 30s)"
+DEFINE_string 'image_tag' 'local' "Tag to use for the images"
+DEFINE_string 'cable_driver' 'libreswan' "Tunneling method for connections between clusters (libreswan, wireguard requires kernel module on host)"
+
+FLAGS "$@" || exit $?
+eval set -- "${FLAGS_ARGV}"
+
+[[ "${FLAGS_globalnet}" = "${FLAGS_TRUE}" ]] && globalnet=true || globalnet=false
+[[ "${FLAGS_service_discovery}" = "${FLAGS_TRUE}" ]] && service_discovery=true || service_discovery=false
+deploytool="${FLAGS_deploytool}"
+deploytool_broker_args="${FLAGS_deploytool_broker_args}"
+deploytool_submariner_args="${FLAGS_deploytool_submariner_args}"
+cluster_settings="${FLAGS_cluster_settings}"
+timeout="${FLAGS_timeout}"
+image_tag="${FLAGS_image_tag}"
+cable_driver="${FLAGS_cable_driver}"
+
+echo "Running with: globalnet=${globalnet@Q}, deploytool=${deploytool@Q}, deploytool_broker_args=${deploytool_broker_args@Q}, deploytool_submariner_args=${deploytool_submariner_args@Q}, cluster_settings=${cluster_settings@Q}, timeout=${timeout}, image_tag=${image_tag}, cable_driver=${cable_driver}, service_discovery=${service_discovery}"
+
+set -em -o pipefail
+
+source ${SCRIPTS_DIR}/lib/debug_functions
+source ${SCRIPTS_DIR}/lib/utils
+source ${SCRIPTS_DIR}/lib/deploy_funcs
+source test/e2eexternal/utils
```

Comment:
Can we move that file to scripts/e2eexternal/utils instead ? 

We only use "test" to carry go code I believe.

Reply:
yeah I was thinking the same thing, ie the bash scripts should be under _scripts_, eg _scripts/e2e-external_.